landing/js/cookie-consent.js

```diff
@@ -40,7 +41,9 @@ $(window).on('load', () => {
     init: () => {
       if (!cookie.checkIfSet('cookie-consent')) {
         cookieMessage.open();
-        bodyElement.showOverlay();
+        if (privacy.length === 0) {
```

Comment:
check if you have to check length or just if(privacyPage) would be enough?

Reply:
It wouldn't be enough because privacyPage is a collection (selecting via class attribute) so we have to check length